### DISCUSSION TITLE
python problem with space usage

### DISCUSSION THREAD

**Discussion by Afifette** [Afifette/mdrlol#13] [Q&A] [answered]
When i start my program it's write "inconscient using of tab or space" pls helpp

**Comment by W3bParserV2** [comment, marked as answer]
### JUST READ ?

**You should have in your code a line starting by a tab and another one starting by 4x space. Replace every tab's by a 4x space or every 4x space by a tab...**

**Reply by Afifette** [reply]
Oh okay i see ty
